config/config.go

```diff
@@ -250,6 +250,9 @@ func Init() error {
 	viper.SetDefault("keybindings.bind_copy_target_url", "c")
 	viper.SetDefault("keybindings.bind_beginning", []string{"Home", "g"})
 	viper.SetDefault("keybindings.bind_end", []string{"End", "G"})
+	viper.SetDefault("keybindings.bind_search", "/")
+	viper.SetDefault("keybindings.bind_next_match", "n")
+	viper.SetDefault("keybindings.bind_prev_match", "p")
```

Comment:
These keybindings need to be added to default-config.toml and to help.go. Don't forget to run `make gen` after doing that as well.

Also please change these keybindings to match Vim/less. See [this comment](https://github.com/makeworld-the-better-one/amfora/issues/36#issuecomment-746541058).